docs/dev/table/sql.md

```diff
@@ -2107,6 +2108,17 @@ VAR_SAMP(value)
         <p>Returns the sample variance (square of the sample standard deviation) of the numeric field across all input values.</p>
       </td>
     </tr>
+
+    <tr>
+      <td>
+          {% highlight text %}
+          COLLECT(value)
+          {% endhighlight %}
+      </td>
+      <td>
+          <p>Returns a multiset of the <i>value</i>s.</p>
```

Comment:
Be more specific about the handling of `null` values. Are they ignored? What is returned if only null values are added (`null` or empty multiset)?

Reply:
done